src/test/e2e/api/admin/feature.e2e.test.ts

```diff
@@ -261,6 +261,34 @@ test('creates new feature toggle with createdBy unknown', async () => {
     });
 });
 
+test('refuses to create a new feature toggle with variant when type is json but value provided is not a valid json', async () => {
```

Comment:
Should we add a test case for valid JSON as well? Not sure if we have an e2e test for the happy path.

Reply:
Just noticed that no "type": json e2e tests were present, I've now added a valid type: json test and one with type: string